### PR TITLE
FeedbackPrompt test (snapshot) 

### PR DESCRIPTION
**Pull request by Butterwell**

### CODE REVIEW

File: src/test/FeedbackPrompt.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+} from 'react-router-dom';
+
+import FeedbackPrompt from '../components/prompts/FeedbackPrompt';
+
+test('A feedback prompt should match the snapshot', () => {
+  let isBlocking = true;
+
+  let openFeedback = () => {
+    this.setState({ feedbackFormRequested: true });
+  };
```
[the remark this answers is not in the document]
Comment (Butterwell) [reply src/test/FeedbackPrompt.test.js]: Okay. Done.

**Comment by knod** [comment]
@ethanbb : If I don't get a chance and you have some free time, can you take a look at what's failing?

**Comment by ethanbb** [comment]
Looks like the `support/netlify` directory doesn't exist - probably just needs another pull from `dev`.

**Comment by knod** [comment]
@Butterwell : Could you update to be in sync with the latest `dev` changes?

**Comment by Butterwell** [comment]
Done. That looks better.

**Pull request merged**